### PR TITLE
Added Cypress tests

### PR DESCRIPTION
**Pull request by mihailradkov**
**Moved existing Cypress tests to test-cypress directory**
- Formatted them according to eslint/editorconfig
- Renamed specification for users import

**Added and configured Cypress for local test execution**
- Added retry plugin and configured 2 retries for failed tests until they are stabilised.
- Renamed test:ci to test:coverage in package.json scripts

**Configured Travis to execute the Cypress tests**

- Added custom GraphDB image with files for server import and added another docker compose which starts the workbench image along with the custom GraphDB image
- This is controlled by NPM test:cypress-ci script
- Configured .dockerignore
- Configured yml in .editorconfig

### CODE REVIEW

File: test-cypress/integration/import/import.server.files.spec.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        cy.chooseServerFilesTab();
+        cy.selectDistinctFileToImport(fileToImport);
+        cy.importChangingSettings().wait(2000);
+        cy.verifyMessageOccurs(messageSuccess);
+    });
+
+    it('Test import Server files successfully with changing base uri', function () {
+        cy.chooseServerFilesTab();
+        cy.selectDistinctFileToImport(fileToImport);
+        cy.importChangingSettings(true);
```
Comment (svilenvelikov) [inline test-cypress/integration/import/import.server.files.spec.js]: We will have hard times with these tests with waits. Definitely we need to refactor them.

File: test-cypress/support/commands.js [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+    cy.get('#import-user > [files-table=""] > [ng-hide="loader"] > .mb-1 > ' +
+        '.form-inline.ng-pristine > .btn-link > .form-check-input')
+        .check({force: true}).should('be.checked');
+
+    cy.get('#wb-import-removeEntries')
+        .click({force: true});
+
+});
+
+let resetStatusOfUploadedFiles = Cypress.Commands.add('resetStatusOfUploadedFiles', () => {
```
Comment (svilenvelikov) [inline test-cypress/support/commands.js]: These selectors are horrible. Who needs to write such a selector when it has an id?!?

File: test-cypress/support/commands.js [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+
+                    cy.get('span[ng-show="currentTabConfig.timeTook"]')
+                        .should('contain', 'Query took');
+                });
+        });
+});
+
+let verifyQueryResultsS = Cypress.Commands.add('verifyQueryResultsS', (numberOfResults) => {
+    let message = numberOfResults ? 'Showing results from 1 to ' + numberOfResults : 'Showing results from 1 to ';
+    cy.get('#yasr-inner .alert.alert-info.no-icon').not("ng-hide")
```
Comment (svilenvelikov) [inline test-cypress/support/commands.js]: I regret I opened this file :)

File: test-cypress/fixtures/example.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "username": "sava",
+  "password": "123"
```
Comment (desislava-hristova-ontotext) [inline test-cypress/fixtures/example.json]: This properties are not used, we can remove the content of the file or the file itself.

**Pull request merged**